nvmeadm/.cargo/config

```diff
@@ -0,0 +1,4 @@
+# we need elevated privileges to run mayastor related tests
```

Comment:
mayastor -> nvmeadm?

Reply:
Fixed in next push: s/copy+paste/copy+paste+read/ ...

Reply:
Fixed